.github/workflows/compliance.yml

```diff
@@ -0,0 +1,114 @@
+        id: get_pr_diff
+        if: contains(steps.label_check.outputs.prlabels, 'Instrumentation') == false
+        run: |
+          echo "Did not find label 'Instrumentation' in '${{steps.label_check.outputs.prlabels}}'"
+          if [ $GITHUB_BASE_REF ]; then
+            # This changes the context of subseqent commands to get the full change for the PR
+            git fetch origin $GITHUB_HEAD_REF:$GITHUB_HEAD_REF
+
+            # Find references to "Sfdo" in the force-app/main/ folder. Filters for the specific class references in later steps.
+            export DIFF=$( git diff-tree origin/$GITHUB_BASE_REF..$GITHUB_HEAD_REF --patch-with-raw -- force-app/main/** | grep 'Sfdo' )
+          fi
+
+          # Assign the "git diff-tree" output to an internal workflow var to use later
+          # Escape newlines (replace \n with %0A) so it's a single line of text
+          echo "::set-output name=diff::$( echo "$DIFF" | sed ':a;N;$!ba;s/\n/%0A/g' )"
+
+          # echo "Diff Output = ${{ steps.get_pr_diff.outputs.diff }}"
+
+      - name: Assign Reviewer if SfdoInstrumentationService is in use
+        id: assign_reviewers_if_instrumentation
+        if: contains(steps.get_pr_diff.outputs.diff, 'SfdoInstrumentationService')
+        uses: SalesforceFoundation/github-script@v4
+        with:
+          script: |
+            github.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              labels: [ "Instrumentation" ]
+            })
+            github.issues.addAssignees({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              assignees: [ "force2b" ]
+            })
+            github.pulls.requestReviewers({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+              reviewers: [ "jstvz" ]
+            })
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: ':warning: @force2b This Pull Request references the "SfdoInstrumentationService" class and must be reviewed by an Instrumentation owner. :warning:'
+            })
+
+      - name: Fail Build if SfdoLogUtils core class is directly referenced
+        id: assign_reviewers_if_core_class
+        if: contains(steps.get_pr_diff.outputs.diff, 'SfdoLogUtils.log(')
+        uses: SalesforceFoundation/github-script@v4
+        with:
+          script: |
+            github.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              labels: [ "Instrumentation" ]
```

Comment:
What happens if the label already exists?

Reply:
Nothing. It just leaves it alone